test/backends/http_test.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, http_server, uhttp_server=None):
         self.dirty = False
         self.requests = 0
 
-        router = http.Router([("/(.*)", self)])
+        router = http.Router([(r"/images/.*/(.*)", self)])
```

Comment:
The real routes are:

        self._server.app = http.Router([
            (r"/images/(.*)/extents", extents.Handler(config, auth)),
            (r"/images/(.*)/checksum/algorithms",
                checksum.Algorithms(config, auth)),
            (r"/images/(.*)/checksum", checksum.Checksum(config, auth)),
            (r"/images/(.*)", images.Handler(config, auth)),
            (r"/info/", info.Handler(config, auth)),
        ])

This is not possible with the current bad way extents is integrated in this
fake handler, since we use one handler, while the real code uses separate
handler for /images/(.*)/extents.

But we can make this simpler using:

    r"/images/(.*)"

and check later:

   if path == "ticket-id/extents":

Reply:
Ok. Different flavors of the same solution.